src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -677,6 +682,8 @@ public OptionData addChoice(@Nonnull String name, @Nonnull String value)
     @Nonnull
     public OptionData addChoices(@Nonnull Command.Choice... choices)
     {
+        if (choices.length == 0)
+            return this;
         if (this.choices == null)
             throw new IllegalStateException("Cannot add choices for an option of type " + type);
         Checks.noneNull(choices, "Choices");
```

Comment:
```suggestion
        if (this.choices == null)
            throw new IllegalStateException("Cannot add choices for an option of type " + type);
        Checks.noneNull(choices, "Choices");
        if (choices.length == 0)
            return this;
```

Current code is vulnerable to NPE if the array is null